mod-orders/schemas/receivingCollection.json

```diff
@@ -30,6 +30,10 @@
                   "description": "The barcode assigned to the item",
                   "type": "string"
                 },
+                "callNumber": {
+                  "description": "The call number assigned to the piece. The call number represents what the book is about and acts like the book's address on the library's shelves or stacks",
```

Comment:
I don't think giving the description of book is correct, it can be any physical item. Please re-use the description from item for a better description